.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: node_js
-node_js: 9
-
+node_js: 10
```

Comment:
What is the reason for node 10 - aren't our extensions running on node 8?

Reply:
but 9 was fine? i updated to 10, since it will also give me an up2date npm, so i don't have to update during install.
-> npm audit

Reply:
I didn't say it was fine and I didn't complain as we talked about doing travis separately, which you did